config/jobs/kubernetes/sig-release/release-branch-jobs/1.17.yaml

```diff
@@ -1200,11 +1191,11 @@ presubmits:
         name: main
         resources:
           limits:
-            cpu: 2
-            memory: 1.2Gi
+            cpu: "2"
+            memory: 1288490188800m
```

Comment:
Can we leave this in the units it was in before?

Reply:
Yeah, this was generated from `./hack/update-generated-tests.sh`.
We'd need to tweak that script to conform.